_includes/header.html

```diff
@@ -25,18 +29,25 @@
         {% endif %}
       {% endif %}
       {% if langmenu %}
-        {% assign posts = site.pages | concat: site.doc | concat: site.qubes-translated | where:'ref', page.ref | sort: 'lang' %}
+        {% assign posts = site.pages | concat: site.doc | concat: site.translated | where:'ref', page.ref | sort: 'lang' %}
         <div class="langdropdown langdropdown-sm">
           <label class="langdropbtn-sm" for="langdropdown-toggle-sm"><img src="/attachment/icons/icons8-translation-64.png" width="24" height="24"></label>
           <input type="checkbox" id="langdropdown-toggle-sm">
           <div class="langdropdown-content" id="langdropdown-dd-sm">
             {% for post in posts %}
-              <a class="c1" href="{{ post.url }}">{{ post.lang }}</a>
+              {% if site.languages contains post.lang %}
+                <a class="c1" href="{{ post.url }}">{{ post.lang }}</a>
+              {% endif %}
```

Comment:
I thought we weren't doing translations for posts?

Reply:
No, it isn't really about news, but about any page. Variable name is confusing here indeed. `posts` is in fact all the pages (docs, and base website pages).